sir_cake/sir_cake/settings.py

```diff
@@ -36,6 +38,10 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+
+    'crispy_forms',
+
+    'seller.apps.SellerConfig',
```

Comment:
If you decided to add seller app related stuff, please also add all of the seller app files. I believe `SellerConfig` is missing right now which means as soon as we merge this PR the main branch will be in a broken state.